rpm/stackdriver-agent.spec

```diff
@@ -42,10 +58,28 @@
 %define bundle_yajl 1
 %endif
 
+%if 0%{?suse_version} > 0
+%define dep_filter 0
+%define bundle_curl 0
+%define java_lib_location /usr/lib64/jvm/java
+%if 0%{?suse_version} < 1500
+%define java_version 1.7.0
+%endif
+%if 0%{?suse_version} >= 1500
+# Yes, SLES really has underscores.
```

Comment:
LOL Without this comment, I would have asked the same question `Are these underscores intentional?`

Reply:
Looks like this comment WAI. :-)